MsCorePkg/MacAddressEmulationDxe/Test/SimpleNetworkProtocolNotifyTest.h

```diff
@@ -0,0 +1,8 @@
+#pragma once
+
+#include "MacAddressEmulationDxeHostTestCommon.h"
+
+VOID
+RegisterSimpleNetworkProtocolNotifyTests (
+    UNIT_TEST_SUITE_HANDLE SuiteHandle
```

Comment:
Can you maybe run a beautify or walk the files for coding standards?  We are trying to watch for even little things like this using a 4 space indent instead of 2.